backend/pkg/database/eventannotations.go

```diff
@@ -0,0 +1,91 @@
+// Copyright © 2021 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package database
+
+import (
+	"context"
+	"gorm.io/gorm"
+	"time"
+)
+
+const (
+	eventAnnotationsTableName = "event_annotations"
+	eventIDColumnName         = "event_id"
+)
+
+var alertKinds = []string{"ALERT"}
+
+type APIEventAnnotation struct {
+	// will be populated after inserting to DB
+	ID         uint   `gorm:"primarykey" faker:"-"`
+	ModuleName string `json:"module_name,omitempty" gorm:"column:module_name;uniqueIndex:api_event_ann_idx_model" faker:"-"`
+	EventID    uint   `json:"event_id,omitempty" gorm:"column:event_id;uniqueIndex:api_event_ann_idx_model" faker:"-"`
+	Name       string `json:"name,omitempty" gorm:"column:name;uniqueIndex:api_event_ann_idx_model" faker:"-"`
```

Comment:
annotations are like key value pairs, where the key is `module_name` + `name` and value is `annotation`. The name in this case is "ALERT" which is a standard annotation for alerts (since alerts are themselves annotations)